### ISSUE TITLE
[DOCS] Mark old rules Bulk API as deprecated

### ISSUE THREAD

**Issue by xcrzx** [elastic/security-docs#1729]
## Description 

**Related to https://github.com/elastic/kibana/pull/126068**

The following Bulk API endpoints were superseded by the new [Bulk Action API](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-action) and are not used anymore:

- [`detection_engine/rules/_bulk_create`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-create)
- [`detection_engine/rules/_bulk_delete`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-delete)
- [`detection_engine/rules/_bulk_update`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-update)

We should mark them as deprecated.

**Comment by xcrzx** [comment]
Hi @jmikell821, correct. We're not deleting these APIs and will still support them for at least 18 months. I think it would make sense to mention the deprecation date (8.2 release date) in docs and clarify that we will delete the APIs only after 18 months.
